tests/gnrc_sock_neterr/main.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @file
+ *
+ * @author      Martine Lenders <m.lenders@fu-berlin.de>
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include "net/sock/udp.h"
+
+#define TEST_PORT               (38664U)
+#define TEST_REMOTE             { 0xfe, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, \
```

Comment:
```suggestion
#define TEST_REMOTE             { 0x20, 0x01, 0x0D, 0xB8, 0x00, 0x00, 0x00, 0x00, \
```

Using an RFC3849 address here should make sure a proper route is never found.

Reply:
I'd rather have both tests TBH, they are different error cases (no route found vs. no [neighboring] host found).